ipywidgets/static/widgets/js/manager.js

```diff
@@ -280,11 +280,26 @@ define([
         /**
          * Handle when a comm is opened.
          */
-        return this.create_model({
-            model_name: msg.content.data.model_name,
-            model_module: msg.content.data.model_module,
-            comm: comm,
-        }).catch(utils.reject("Couldn't create a model.", true));
+        // For < 4.x
+        var model;
+        if (msg.content.data.model_name) {
```

Comment:
I know deprecation warnings can be a PITA to keep track of, but I think they are much less of a PITA for us than it is for our friends to migrate without them.